src/common/toast/hdsToast.tsx

```diff
@@ -9,26 +9,32 @@ import i18n from '../../locales/i18n';
 import { HDSToastContainerId } from './HDSToastContainer';
 
 interface HDSToastArgs {
+  autoDismiss?: boolean;
   type: DismissableNotificationProps['type'];
   labelText: string;
   text: string;
   toastId?: string;
   translated?: boolean;
 }
 
-interface DismissableNotificationWrapperProps extends HDSToastArgs {
+interface NotificationWrapperProps extends HDSToastArgs {
   toastId: string;
   translated: boolean;
 }
 
-const DismissableNotificationWrapper = ({
-  type,
-  labelText,
-  text,
-  toastId,
-  translated,
-}: DismissableNotificationWrapperProps) => {
+const NotificationWrapper = ({ autoDismiss, type, labelText, text, toastId, translated }: NotificationWrapperProps) => {
   const { t } = useTranslation();
+  const autoDismissTime = 3000;
+
+  useEffect(() => {
+    if (autoDismiss) {
+      const timer = setTimeout(() => {
```

Comment:
`react-toastify` seems to already support `autoClose` should we use that instead?

Reply:
See https://fkhadra.github.io/react-toastify/autoClose

Reply:
This _sounds_ like a good idea, and I'm trying it out right now. However, `autoClose: 3000` does not actually seem to be doing anything, not even if I specify it for the toast container.

Reply:
Yeah, did a quick research on this and it seems that the prop got broken on React update. No easy workaround, so let's go with this.